builds.yml

```diff
@@ -31,6 +31,9 @@ buildTypes:
       - KEYRING_SNAPS_REGISTRY_URL: https://metamask.github.io/keyring-snaps-registry/prod/registry.json
     # Main build uses the default browser manifest
     manifestOverrides: false
+    # Build name used in multiple user-readable places
+    # eg. eip6963
+    buildNameOverride: MetaMask
```

Comment:
it's awkward because the name defined in manifest.json is also user-readable, but this buildNameOverride does not override it. 

https://github.com/MetaMask/metamask-extension/blob/fc724d6d974c239c9fc38d805d89cb5f3e00c9e6/app/build-types/mmi/manifest/chrome.json#L3

https://github.com/MetaMask/metamask-extension/blob/fc724d6d974c239c9fc38d805d89cb5f3e00c9e6/app/build-types/flask/manifest/chrome.json#L3

This property is still confusing given its name and the current manifest.json base files for each build type

Reply:
No need to change anything here, just wanted to point out that it could potentially still be confusing to futures devs tracking this down